src/BootKeyboard/BootKeyboard.cpp

```diff
@@ -196,15 +196,29 @@ uint8_t BootKeyboard_::getLeds(void) {
 }
 
 uint8_t BootKeyboard_::getProtocol(void) {
-  return protocol;
+  if (boot_protocol_) {
+    return HID_BOOT_PROTOCOL;
+  } else {
+    return HID_REPORT_PROTOCOL;
+  }
```

Comment:
This reports the protocol based on the new `boot_protocol_` variable instead of `protocol`. This is ultimately what controls which type of report is sent, in the HID adaptor code. See [this line in Kaleidoscope-HIDAdaptor-KeyboardioHID](https://github.com/keyboardio/Kaleidoscope-HIDAdaptor-KeyboardioHID/blob/f3803685a901698e1bbd0dc3f9c5a637415df62e/src/kaleidoscope/hid.cpp#L33) for details.